### PR TITLE
Update developer guide, add PPP and fix bugs

### PR DESCRIPTION
**Pull request by olivercheok20**

### CODE REVIEW

File: src/main/java/seedu/address/ui/MainWindow.java [hunk 1]
```diff
@@ -240,8 +240,6 @@ private void showExpandPolicy(CommandResult commandResult) {
      */
     private boolean usesRightPane(CommandResult commandResult) {
         return commandResult.isDisplay()
-            || commandResult.isExpandPerson()
-            || commandResult.isExpandPolicy()
             || commandResult.isListHistory();
```
Comment (chaitanyabaranwal) [inline src/main/java/seedu/address/ui/MainWindow.java]: Just curious, but is there no other way to fix this issue?

File: src/main/java/seedu/address/model/person/NameContainsKeywordsPredicate.java [hunk 2]
```diff
@@ -18,7 +16,7 @@ public NameContainsKeywordsPredicate(List<String> keywords) {
     @Override
     public boolean test(Person person) {
         return keywords.stream()
-                .anyMatch(keyword -> StringUtil.containsWordIgnoreCase(person.getName().fullName, keyword));
+                .anyMatch(keyword -> person.getName().fullName.toLowerCase().contains(keyword.toLowerCase()));
```
Comment (chaitanyabaranwal) [inline src/main/java/seedu/address/model/person/NameContainsKeywordsPredicate.java]: Maybe you can change this in the `StringUtil::containsWordIgnoreCase` function to maintain the same level of abstraction?

**Comment by chaitanyabaranwal** [comment]
This PR, other than the comments I made, has Yiyin's commits (which are already merged into the `master` branch), making the commit history a bit messy. Would be great if you can rebase again from the master branch to clean up the commit history!

File: src/main/java/seedu/address/model/person/NameContainsKeywordsPredicate.java (hunk 2 above)
Comment (olivercheok20) [reply src/main/java/seedu/address/model/person/NameContainsKeywordsPredicate.java]: Fixed in the latest commit :)

**Pull request closed without merging**